data/itemtooltips.lua

```diff
@@ -0,0 +1,43 @@
+local addonName = ... ---@type string
+
+---@class BetterBags: AceAddon
+local addon = LibStub('AceAddon-3.0'):GetAddon(addonName)
+
+---@class itemTooltips: AceModule
+
+local itemTooltips = addon:NewModule('ItemTooltips')
+
+---@class Events: AceModule
+local events = addon:GetModule('Events')
+
+function itemTooltips:OnInitialize()
+end
+
+function itemTooltips:OnEnable()
+  self:Update()
+end
+
+function itemTooltips:Update()
+ -- todo: Do we need to deal with TOOLTIP_DATA_UPDATE or does itemMixin:IsItemDataCached() gather tooltips?
+end
+
+---@param tooltipData tooltipData
+---@return Enum.TooltipDataItemBinding|nil
+function itemTooltips:GetItemBinding(tooltipData)
```

Comment:
While we're here, do we want to just extract everything we can from the tooltip, and add it to the data properties?

Reply:
This is exactly why I created a module for this. Binding is just one possibility. I'd have to ponder other use cases not covered by the item api